plugins/hadoop-authorization-sentry-default/sentry-client/src/main/java/com/thinkbiganalytics/datalake/authorization/client/SentryClientConfig.java

```diff
@@ -42,6 +44,20 @@
     private DataSource dataSrouce;
     private JdbcTemplate sentryJdbcTemplate;
     private String sentryGroups;
+    private String authorizationGroupType;
+    private String linuxGroupFilePath;
+
+
+    /**
+     * LDAP Connection Parameter 
+     */
+    private String ldapURL;
+    private String ldapAdmin;
+    private String ldapAdminPassword;
+    private LdapContextSource LdapContextSource;
+    private String LdapGroupDnPattern;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this field "LdapGroupDnPattern" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00116)